app/src/main/res/values-ar/strings.xml

```diff
@@ -0,0 +1,358 @@
+  <string name="msg_yesno_def_message">هل أنت واثق؟</string> 
+  <string name="msg_view_deletion_failed">تعذر حذف العرض.</string> 
+  <string name="msg_empty_ui_layout">لم يتم إضافة وجهات النظر. انقر للإضافة.</string> 
+  <string name="msg_no_attr_format">لا يوجد تنسيق سمة متاح لهذه السمة.</string> 
+  <string name="msg_generate_xml_failed">فشل إنشاء كود XML</string> 
+  <string name="title_generating_xml">توليد كود XML</string> 
+  <string name="title_code_generation_failed">فشل إنشاء الكود</string> 
+  <string name="msg_code_generation_failed">فشل إنشاء رمز XML. هل ترغب في الخروج على أي حال؟</string> 
+  <string name="msg_invalid_designer_result">قام مصمم واجهة المستخدم بإرجاع نتيجة غير صالحة</string> 
+  <string name="msg_swipe_up">مرر سريعًا لأعلى لإنشاء مخرجات وسجلات والمزيد.</string> 
+  <string name="msg_ide_crashed">تحطمت AndroidIDE</string> 
+  <string name="msg_report_crash">الرجاء فتح مشكلة على github باستخدام تتبع المكدس التالي:</string> 
+  <string name="msg_crash_report_hint">انقر فوق الزر أدناه لنسخ السجل وفتح صفحة المشكلات.</string> 
+  <string name="msg_copy_report_action">نسخ وتقرير</string> 
+  <string name="msg_reading_bootstrap">قراءة أرشيف التمهيد ...</string> 
+  <string name="msg_bootstrap_installation_error">فشل تثبيت Bootstrap مع الاستثناء التالي.</string> 
+  <string name="title_translations">الترجمات</string> 
+  <string name="title_compiler_module_install">تركيب وحدة المترجم</string> 
+  <string name="msg_compiler_module_install">جاري تثبيت وحدة المترجم ، الرجاء الانتظار ...</string> 
+  <string name="msg_compiler_module_install_failed">فشل تثبيت وحدة المترجم. كان الخطأ: \n٪ s</string> 
+  <string name="msg_compiler_module_installed">تم تثبيت وحدة المترجم بنجاح!</string> 
+  <string name="action_expand_selection">قم بتوسيع التحديد</string> 
+  <string name="idepref_java_matchLower_title">إتمام المباراة بأحرف صغيرة</string> 
+  <string name="idepref_editor_category_common">مشترك</string> 
+  <string name="idepref_editor_category_java">جافا</string> 
+  <string name="title_files_unsaved">الملفات غير المحفوظة</string> 
+  <string name="msg_files_unsaved">لم يتم حفظ بعض الملفات. هل ترغب في حفظها قبل الإغلاق؟ الملفات غير المحفوظة هي: \n٪ s</string> 
+  <string name="idepref_editor_autoSave_title">احفظ قبل إغلاق الملفات</string> 
+  <string name="idepref_editor_autoSave_summary">احفظ الملفات قبل إغلاقها. هذه ميزة تجريبية. من المتوقع أن تقوم بحفظ الملفات بنفسك.</string> 
+  <string name="idepref_java_matchLower_summary">في حالة التمكين ، ستؤدي كتابة أحرف صغيرة أو أحرف كبيرة إلى اقتراح أسماء فئة بالإضافة إلى أعضاء النطاق.</string> 
+  <string name="idepref_editor_ligatures_title">وصلات الخط</string> 
+  <string name="idepref_editor_ligatures_summary">تمكين / تعطيل وصلات الخط</string> 
+  <string name="yes">نعم</string> 
+  <string name="no">رقم</string> 
+  <string name="msg_apk_install_intent_failed">تعذر بدء النية لتثبيت APK</string> 
+  <string name="msg_viewaction_add_attr">اضف ميزة</string> 
```

Comment:
This translates to `Add feature`. It should be `Add attribute`.

Reply:
Feature is same as attribute on Arabic